ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -223,9 +223,10 @@ values
     ;
 
 selectItem
-    : expression (AS? identifier)?  #selectSingle
-    | identifier '.' ASTERISK       #selectAll
-    | ASTERISK                      #selectAll
+    : expression (AS? identifier)?                       #selectSingle
+    | base=primaryExpression STRUCT_FIELD_REF ASTERISK   #selectStructAll
```

Comment:
Separate question... how is whitespace handled?

Should `primaryExpress -> *` work?

Reply:
ANTlr handles that for us. I think it ignores spaces. I tested the ` -> ` and it works.